assets/runtime/config/gitlabhq/cable.yml

```diff
@@ -0,0 +1,14 @@
+# This is a template taken from here:
+# https://gitlab.com/gitlab-org/gitlab/-/blob/master/config/cable.yml.example
+development:
+  adapter: redis
+  url: redis://localhost:6379
+  channel_prefix: gitlab_development
+test:
+  adapter: redis
+  url: redis://localhost:6379
+  channel_prefix: gitlab_test
```

Comment:
If the definition of the Redis connection for the environments 'development' and 'test' is not needed, then I would suggest to either use other "default values" (like
`url: redis://:fakepass@fake.redis.actioncable.com:1234` and `channel_prefix: gitlab_fake_development`) or these environmental setups should not be defined at all.

Reply:
Yes at the beginning I haven’t defined them, but then I saw this file: https://github.com/sameersbn/docker-gitlab/blob/09e76bb2e7d4c805bfc2cb70cb1b0ca89cd60216/assets/runtime/config/gitlabhq/resque.yml and replicated the same behavior. Anyway it is quite harmless to keep the other two, it’s just for consistency with other files.

Reply:
Okay, that sounds reasonable. Have you already tested this version and can you confirm that the error in the `production.log` file no longer occurs?

Reply:
@sachilles yes sure, using it in production now, it works like a charm.

Reply:
That sounds good. Thanks for your contribution! A patch release will follow soon.

Reply:
The bug affects also 14.4 and 14.5 @sachilles 

thanks for merging!